### PR TITLE
Update env-configuration.md

### PR DESCRIPTION
**Pull request by silentoplayz**
Update Open WebUI's list of environment variables (v0.3.20 -> v0.5.11). My apologies for the two separate initial commits rather than one to help compare side-by-side. I'm still learning GitHub Desktop.

This PR adds environment variables accessed by Open WebUI, but were not previously documented. I also tried to sort the environment variables the best I could, so there's a bunch of new categories.

Future improvements to Open WebUI's list of environment variables can include:
- [x] Distinguishing between persistent and non-persistent environment variables.
- [ ] Listing available options (Sorta Hard) and default values (should mostly be complete) for restricted environment variables where necessary.
- [ ] Identifying incompatible variable combinations and provide warnings or notes. (Hard)
- [ ] Reviewing each environment variable and fixing any misinformation to improve correctness. (Also Hard)

Before:
image

After:
image

### CODE REVIEW

**Review by 0xThresh — APPROVED** [review]
Thank you for such a great effort on getting the docs updated!

File: docs/getting-started/advanced-topics/env-configuration.md
Comment (tjbck) [inline docs/getting-started/advanced-topics/env-configuration.md:1]: We might want to indicate which ones are `persistentConfig` and which ones are just env var.

Comment (silentoplayz) [reply docs/getting-started/advanced-topics/env-configuration.md:1]: Working on this today and should have a new commit pushed out soon!

**Comment by silentoplayz** [comment]
Is this ready to be merged? In the future, I believe the env list page could be updated if necessary. This already is a huge bump to the details on the page.

**Pull request merged**